### PR TITLE
Sort output of lichking list.

### PR DESCRIPTION
**Pull request by meltinglava**
We use itertools as it has `sorted`, `join` and `sorted_by_key`.
All of the places that had `collect`, itertools does the same in the background. And it cleans up nicely.

fixes: #59 

bonus pun:
We have now purged the unorder in Stratholme

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #![allow(unknown_lints)] // for clippy
 
+#[macro_use] extern crate itertools;
```
Comment (Nemo157) [inline src/main.rs]: You can remove the `#[macro_use]` here, that will fix the clippy CI failure.

Comment (meltinglava) [reply src/main.rs]: But this line is needed. It does not compile when i remove it. Fix is probably to move all the code to new edition and fix all warnings. I can do that, thow it feels like it should be a new PR

Comment (Nemo157) [reply src/main.rs]: The `extern crate` is still needed, but it doesn’t need the `#[macro_use]` on it.

**Comment by Nemo157** [comment]
bors r+

Thanks!

**Pull request merged**